KsApi/px/MockPerimeterXTypes.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import PerimeterX
+
+extension PerimeterXClient {
+  static var mock: PerimeterXClient = PerimeterXClient(
+    manager: MockPerimeterXManager(),
+    dateType: ApiMockDate.self
+  )
+}
+
+struct MockPerimeterXBlockResponse: PerimeterXBlockResponseType {
+  var blockType: PXBlockType
+
+  var type: PXBlockType {
+    return self.blockType
+  }
+}
+
+final class MockPerimeterXManager: PerimeterXManagerType {
+  var headers: [AnyHashable: Any] = ["PX-AUTH-TEST": "foobar"]
```

Comment:
I think we can make these variables private given the accessor methods below?

Reply:
These are just exposed if we want to manipulate in tests.